README.md

```diff
@@ -1 +1,32 @@
-# NF1_SchwannCell_data
\ No newline at end of file
+# NF1 Schwann Cell Data Project
+
+## Data
+
+The data used in this project is a modified Cell Painting assay on [schwann cells](https://www.ncbi.nlm.nih.gov/books/NBK544316/) from patients with [Neurofibromatosis type 1 (NF1)](https://medlineplus.gov/genetics/condition/neurofibromatosis-type-1/). 
+In this modified Cell Painting, there are three channels:
+
+- `DAPI` (Nuclei)
+- `GFP` (Endoplasmic Reticulum)
+- `RFP` (Actin)
+
```

Comment:
Can you add an example image?? that would be so nice :)

Reply:
Done deal!